Sources/PointFree/StripeWebhooks.swift

```diff
@@ -94,11 +94,14 @@ private func handleFailedPayment(
   -> IO<Conn<ResponseEnded, Data>> {
 
     return Current.stripe.fetchSubscription(conn.data)
+      .withExcept(notifyError(subject: "Stripe Hook failed: Couldn't find stripe subscription."))
       .flatMap(Current.database.updateStripeSubscription)
       .mapExcept(requireSome)
+      .withExcept(notifyError(subject: "Stripe Hook failed: Couldn't find updated subscription."))
```

Comment:
it's the `requireSome`'s that are turning the error into `Prelude.Unit`:

https://github.com/pointfreeco/pointfreeco/blob/a57a4c8b1aa4dd183ab78cb566024fef0e13b65d/Sources/PointFreePrelude/Either.swift#L18